tests/ltp/patches/fix_chroot_chroot02.patch

```diff
@@ -0,0 +1,35 @@
+commit 9dacbeffecd080404ddcc71e33dd4698c5656818
```

Comment:
Can you remove this header except for the description starting with "Removed exit code f.."?